### PR TITLE
flash memory impl

### PR DESCRIPTION
**Pull request by x37v**

### CODE REVIEW

File: src/flash.rs [hunk 1]
```diff
@@ -0,0 +1,313 @@
+//! IS25LP064A: 64Mbit/8Mbyte flash memory
+//!
+//! https://www.issi.com/WW/pdf/25LP032-64A-B.pdf
+//!
+
+// 133Mhz
+// bank 1
+//  pin_group[DSY_QSPI_PIN_IO0] = dsy_pin(DSY_GPIOF, 8);
```
Comment (mtthw-meyer) [inline src/flash.rs]: Dead code?

Comment (x37v) [reply src/flash.rs]: it is the c++ source that I used to get the info for how the pins were mapped.. i've ended up finding that much easier to grok than the other info around.. I'll ditch it though

Comment (mtthw-meyer) [reply src/flash.rs]: If it's useful to have you can leave it, make it clear it's a comment on something.

Comment (x37v) [reply src/flash.rs]: nah, that info is all encoded in the Rust code now.. ditched

**Pull request merged**